app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js

```diff
@@ -10,31 +10,25 @@ export default class extends Controller {
   @tracked selectedSidebarTagNames = [];
 
   @action
-  tagUpdated(tagNames) {
-    this.selectedSidebarTagNames = tagNames;
-    this.model.set("sidebar_tag_names", tagNames);
-    this.saved = false;
-  }
+  save() {
+    const initialSidebarCategoryIds = this.model.sidebarCategoryIds;
+    const initialSidebarTagNames = this.model.sidebarTagNames;
 
-  @action
-  categoryUpdated(categories) {
-    this.selectedSiderbarCategories = categories;
-    this.model.set("sidebarCategoryIds", categories.mapBy("id"));
-    this.saved = false;
-  }
+    this.model.set("sidebar_tag_names", this.selectedSidebarTagNames);
```

Comment:
I think `sidebar_tag_names` here and in the catch should be `sidebarTagNames` ?

Reply:
This is correct, alittle confusing because I have an alias for `sidebar_category_ids` to be `sidebarCategoryIds` but `sidebarTagNames` is a computed property.